buildpipeline/Core-Setup-Linux.json

```diff
@@ -88,8 +88,8 @@
         "definitionType": "task"
       },
       "inputs": {
-        "scriptPath": "build.sh",
-        "args": "$(BuildArguments)",
+        "scriptPath": "scripts/dockerrun-as-current-user.sh",
+        "args": "-t --rm --sig-proxy=true --name $(DockerContainerName) -v $(Build.SourcesDirectory):/opt/code -w /opt/code -e CONNECTION_STRING -e PUBLISH_TO_AZURE_BLOB $(DockerImageName) /bin/bash -c \"HOME=/opt/code; git clean -X -d -f; ./build.sh $(BuildArguments)\"",
```

Comment:
I don't think `-e CONNECTION_STRING -e PUBLISH_TO_AZURE_BLOB` is enough for all builds, e.g. `REPO_ID`. I think it would make sense to change/add to the build scripts to work with a specified image name to avoid duplicating the environment variable names [in dockerrun](https://github.com/dotnet/core-setup/blob/53d6b3c1488ffd67888b86a5feda12e3a8a29c84/scripts/dockerrun.sh#L138-L154). Then the other build defs could consolidate their `-e` strings too.